src/fidesops/service/connectors/base_connector.py

```diff
@@ -29,6 +30,11 @@ class BaseConnector(Generic[DB_CONNECTOR_TYPE], ABC):
 
     def __init__(self, configuration: ConnectionConfig):
         self.configuration = configuration
+        # If Fidesops is running in test mode, it's OK to show
+        # parameters inside queries for debugging purposes. By
+        # default we assume that Fidesops is not running in test
+        # mode.
+        self.hide_parameters = not os.getenv("TESTING") == "True"
```

Comment:
I feel like if we're going to use this everywhere (and I think it's going to get used a lot, we've already discussed other uses like the hot-reloading flag, maybe log masking as well) we should have a place where we set a global application state and not refer to this all over the codebase with `os.getenv("TESTING")`